src/discord.re

```diff
@@ -1,29 +1,31 @@
-type client;
-
-type message;
-
-type user;
-
-type channel;
-
-external createClient : unit => client = "Client" [@@bs.module "discord.js"] [@@bs.new];
-
-external login : client => string => unit = "" [@@bs.send];
-
-external onReady : client => _ [@bs.as "ready"] => (unit => unit) => unit = "on" [@@bs.send];
-
-external onMessage : client => _ [@bs.as "message"] => (message => unit) => unit =
-  "on" [@@bs.send];
-
-external getContent : message => string = "content" [@@bs.get];
-
-external getChannel : message => channel = "channel" [@@bs.get];
-
-external getType : channel => string = "type" [@@bs.get];
-
-external getAuthor : message => option user =
-  "author" [@@bs.get] [@@bs.return null_undefined_to_opt];
-
-external isBot : user => bool = "bot" [@@bs.get];
-
-external reply : message => string => unit = "" [@@bs.send];
+module Channel = {
+  type t;
+  /* May change later to TextChanngel.t, VoiceChannel.t, and so on */
+  type cType =
+    | Text
+    | Voice
+    | DM
+    | Group;
```

Comment:
There are four types in discord.js
https://discord.js.org/#/docs/main/stable/class/Channel